lib/modules/manager/github-actions/__fixtures__/workflow_4.yml

```diff
@@ -0,0 +1,19 @@
+jobs:
+  test:
+    steps:
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # 1.2.3
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # 1.2
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # 1
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # v1.2.3
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # v1.2
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # v1
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # @v2.1.0
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # pin @v2.1.0
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # tag=v2.1.0
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97  #   v2.1.0
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 #v2.1.0
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 #v2.1.0
+      - uses: actions/checkout@1e204e # v2.1.0
+      - uses: actions/checkout@01aecc#v2.1.0
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # comment containing 2.1.0
+      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # v2.1.0 additional comment
```

Comment:
Can you use different sha for non working values, so it's easier to see wrong matches

Reply:
I've included all particular strings in the snapshot as the `replaceString` value, so it's should be quite easy

Reply:
yes, but there are cases which shouldn't match and if we change the sha for them, it more easily to see if it will be wrongly matched.

Reply:
```suggestion
      - uses: actions/checkout@689fcce700ae7ffc576f2b029b51b2ffb66d3abd # comment containing 2.1.0
      - uses: actions/checkout@689fcce700ae7ffc576f2b029b51b2ffb66d3abd # v2.1.0 additional comment
```